documentation/migrate-from-silverlight/compile-with-opensilver.md

```diff
@@ -66,17 +66,17 @@ Visit the "[Troubleshooting](../troubleshooting/common-issues-and-solutions.md)"
 
 It is possible that OpenSilver doesn't have required class or method implemented yet. The temporary solution for compiling the project would be to add empty classes/methods in OpenSilver "Empty Namespaces" folder and use that .dll instead.
 
 #### In XAML files:
 
 As far as XAML files are concerned, since compiler directives do not work in XAML, we recommend these 2 approaches to make changes to the XAML code without breaking the original Silverlight application:
-- If it is a small change, try to do it in the C# code-behind file instead of the XAML file. To do so, first add x:Name="SOME_NAME" (in XAML) to the control that you wish to modify, then make the change in the constructor in the C# code-behind, and use #if OPENSILVER compiler directive to make sure that the change does not break the original Silverlight version of the application.
-- If the change is extensive or if it cannot be done in the C# code-behind, we recommend you to create a copy of the XAML file, and reference that copy in your OpenSilver project, while the original Silverlight application references the original file. For example, if you need to make a change to "App.xaml", you should create a copy named for example "App.OpenSilver.xaml", and have the OpenSilver project  reference that file instead of "App.xaml". Note: the associated C# code-behind file ("App.xaml.cs") can still be shared between the original and migrated projects.
+- If it is a small change, try to do it in the C# or VB.NET code-behind file(as required) instead of the XAML file. To do so, first add x:Name="SOME_NAME" (in XAML) to the control that you wish to modify, then make the change in the constructor in the C# or VB.NET code-behind, and use #if OPENSILVER compiler directive to make sure that the change does not break the original Silverlight version of the application.
+- If the change is extensive or if it cannot be done in the C# or VB.NET code-behind, we recommend you to create a copy of the XAML file, and reference that copy in your OpenSilver project, while the original Silverlight application references the original file. For example, if you need to make a change to "App.xaml", you should create a copy named for example "App.OpenSilver.xaml", and have the OpenSilver project  reference that file instead of "App.xaml". Note: In C# migration, the associated C# code-behind file ("App.xaml.cs") can still be shared between the original and migrated projects. In VB.NET migration, the associated VB.NET code-behind file will be ("App.xaml.vb") can be shared between the original and migrated projects.
```

Comment:
```suggestion
- If the change is extensive or if it cannot be done in the C# or VB.NET code-behind, we recommend you to create a copy of the XAML file, and reference that copy in your OpenSilver project, while the original Silverlight application references the original file. For example, if you need to make a change to "App.xaml", you should create a copy named for example "App.OpenSilver.xaml", and have the OpenSilver project  reference that file instead of "App.xaml". Note: the associated C# or VB.NET code-behind file ("App.xaml.cs/App.xaml.vb") can still be shared between the original and migrated projects.
```